ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -427,6 +427,21 @@ tests:
     test:
     - ref: openshift-e2e-test
   timeout: 6h0m0s
+- always_run: false
```

Comment:
I think we can make this a `run_if_changed: aws` so it always runs for aws jobs.